retrybp/filters.go

```diff
@@ -139,6 +140,26 @@ func RetryableErrorFilter(err error, next retry.RetryIfFunc) bool {
 	return next(err)
 }
 
+// BreakerErrorFilter is a Filter implementations that retries circuit-breaker
+// errors from gobreaker/breakerbp.
+//
+// It should only be used when you are using circuit breaker,
```

Comment:
I feel like it can be used even if you aren't, it just won't ever match.  Perhaps

```
This is only useful if you are using...
```

Reply:
here the "should only" is emphasizing on the "and have proper backoff policy set" part. I just added the line breaker after comma because this is like markdown that it doesn't care about line breakers so I prefer to put line breakers after commas and periods instead of purely based on line length.

the reason it should only be used with proper backoff policy set is that without proper backoff policy, it's highly likely the retries will just fail and not really doing anything.